main/zoe/guide/contracts/README.md

```diff
@@ -3,15 +3,24 @@
 <Zoe-Version/>
 
 ::: tip Out-of-date status
-Zoe's master branch is currently an Alpha release candidate. This doc is in the process of being updated, and should be current with the release candidate in another few days. What you see here is out of date. We apologize for any inconvenience this may cause.
+Zoe's master branch is currently an Alpha release candidate. This doc is in the process of
+being updated, and should be current with the release candidate in another few days. What you
+see here is out of date. We apologize for any inconvenience this may cause.
 :::
 
-While Zoe provides the means to build custom smart contracts, there is a good chance you will want to use one that follows a commonly-used structure. Therefore, we currently provide five pre-built contracts that can be imported and run on Zoe:
+While Zoe provides the means to build custom smart contracts, there is a good chance you will
+want to use one that follows a commonly-used structure. Therefore, we currently provide five
```

Comment:
We're up to at least six contracts with barter-exchange.  Change five to however many we're going to end up with.